src/managers/channels.ts

```diff
@@ -66,4 +76,93 @@ export class ChannelsManager extends BaseManager<ChannelPayload, Channel> {
         .catch((e) => reject(e))
     })
   }
+
+  async sendMessage(
+    channel: string | TextChannel,
+    content?: string | AllMessageOptions,
+    option?: AllMessageOptions,
+    reply?: Message
```

Comment:
I think adding reply to message option will be better

Reply:
done